package.json

```diff
@@ -25,6 +25,7 @@
     "watch-test": "mocha --require ts-node/register ./test/**/*-test.[tj]s --watch",
     "autorest:clean": "rimraf src/util/apis/generated",
     "autorest": "npm run autorest:clean && node scripts/autorest.js",
+    "autorest:gen": "node --max-old-space-size=16384 node_modules/bin/autorest --debug swagger/readme.md",
```

Comment:
Does this work for you? Both on my Windows and macOS machines I have a .bin folder in node_modules instead of a bin one, which contains autorest. I think this should be `node_modules/.bin/autorest` just like in the autorest.js file. It still won't work on Windows then because there it's called autorest.cmd instead.
Maybe use a script just like the `autorest` command above?

Reply:
Ooops, bad copy & paste.
Is the `.cmd` file actually javascript? Can you execute it using `node node_modules/.bin/autorest.cmd`?

Reply:
That doesn't work. But the autorest file is also not JavaScript, but bash code. These are the contents of autogen.cmd:

```cmd
@ECHO off
SETLOCAL
CALL :find_dp0

IF EXIST "%dp0%\node.exe" (
  SET "_prog=%dp0%\node.exe"
) ELSE (
  SET "_prog=node"
  SET PATHEXT=%PATHEXT:;.JS;=;%
)

"%_prog%"  "%dp0%\..\autorest\dist\app.js" %*
ENDLOCAL
EXIT /b %errorlevel%
:find_dp0
SET dp0=%~dp0
EXIT /b
```

Reply:
Okay thanks. Will just call `app.js` instead then.
This new script is really just a convenience anyway, allowing you to run `autorest` only without the re-download of the json file.